setup.py

```diff
@@ -6,7 +6,7 @@
 here = path.abspath(path.dirname(__file__))
 
 # get the long description from the README file
-with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+with open(path.join(here, 'README.rst'), encoding='utf-8') as f:
```

Comment:
@dsmits this fucked up the install as a python package

Reply:
:sweat_smile: